src/test/test_worker.ts

```
@@ -1717,11 +1712,7 @@ describe('workers', function() {
 
       await activating;
 
-      try {
-        await job.retry();
-      } catch (err) {
-        expect(err.message).to.equal('Retried job not failed');
-      }
+      await expect(job.retry()).to.be.rejectedWith('Retried job not failed');
```

I recommend expect errors in this way, if not we could get silent failures in our tests as in this case